app/backend/src/tests/test_fixtures.py

```diff
@@ -210,39 +197,100 @@ def drop_database() -> None:
                 "DROP SCHEMA IF EXISTS public CASCADE;"
                 "DROP SCHEMA IF EXISTS logging CASCADE;"
                 "DROP EXTENSION IF EXISTS postgis CASCADE;"
-                "CREATE SCHEMA public;"
-                "CREATE SCHEMA logging;"
+                "CREATE SCHEMA IF NOT EXISTS public;"
+                "CREATE SCHEMA IF NOT EXISTS logging;"
                 "CREATE EXTENSION postgis;"
                 "CREATE EXTENSION pg_trgm;"
                 "CREATE EXTENSION btree_gist;"
             )
         )
 
 
-def recreate_database():
+@contextmanager
+def autocommit_engine(url: str):
+    engine = create_engine(
+        url,
+        isolation_level="AUTOCOMMIT",
```

Comment:
Why do you want an autocommit engine? Could you add a docstring here please?

If I understand correctly it's so that you can run things like `create database`.

Reply:
Exactly that. Added a comment